lib/Backend/arm64/LowerMD.cpp

```diff
@@ -6682,9 +6682,16 @@ LowererMD::EnsureEHEpilogLabel()
         return this->m_func->m_epilogLabel;
     }
 
+    IR::Instr *exitInstr = this->m_func->m_exitInstr;
```

Comment:
I see you closed #4224 without merging. This function looks very similar to EnsureEpilogLabel. What's the intention here?

Reply:
#4224 ended up not working in all cases for ARM64. There were some cases where it would added by GenerateJumpToEpilogForBailOut during Lower, removed by peeps, and then references by EnsureEpilogLabel in FinalLower.